msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -1121,9 +1146,26 @@ internal void ClearAdalCache()
             LegacyCachePersistence.WriteCache(AdalCacheOperations.Serialize(dictionary));
         }
 
-        internal void ClearMsalCache()
-        {
-            TokenCacheAccessor.Clear();
+
+        internal void ClearMsalCache()
+        {
+            TokenCacheAccessor.Clear();
+        }
+
+
+        internal SortedSet<string> ParseScopesForAdfsToken(SortedSet<string> scopes)

```

Comment:
Not sure that ADFS v4 scope support was tested comprehensively, i did not manage to get at for resource from Lab ADFS V4 env

Reply:
What was the request you sent? There are some limitations regarding adfs supportability (only a single resource is allowed)

Reply:
yes, i have used adfs v4